src/lib/y2storage/yaml_writer.rb

```diff
@@ -440,5 +441,70 @@ def yaml_btrfs_subvolume(subvol)
       content["nocow"] = "true" if subvol.nocow?
       { "subvolume" => content }
     end
+
+    # Return YAML for any unsupported objects in a devicegraph.
+    # Right now this is limited to block devices; this might change in the
+    # future.
+    #
+    # @param devicegraph [Devicegraph]
+    # @return [Hash]
+    def yaml_unsupported(devicegraph)
+      content = []
+      unsupported_devices(devicegraph).each { |device| content << yaml_unsupported_device(device) }
+      return nil if content.empty?
+
+      { "unsupported" => content }
+    end
+
+    # Return YAML for one unsupported device.
+    #
+    # @param  device [Device]
```

Comment:
Maybe worth mentioning that it should be a device responding to `#name`. Which, BTW, in the usual Ruby/Yardoc notation can be specified as
```ruby
# @param device [#name]
```

Reply:
Oops, I was pretty sure that _name_ is defined for all devices. If that's not the case, I'll rather add a `if respond_to?(:name)`.